app/settings/surveys/attribute-editor.html

```diff
@@ -35,6 +35,52 @@
           <svg class="iconic" role="img">
             <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/img/iconic-sprite.svg#trash"></use>
           </svg>
+            Only people with permission to edit responses will be able to see responses to this field.
+        </p>
+        <div class="toggle-switch">
+           <input class="tgl" id="switchprivateresponse" type="checkbox" ng-model="editAttribute.response_private">
+           <label class="tgl-btn" for="switchprivateresponse"></label>
+        </div>
+       </div>
+       <div class="form-field switch" ng-if="canDisplayDefaultValue()">
```

Comment:
You only need one <label> and <input> element here. You can define the placeholder and type in the directive and use that variable here to make it change depending on what type of field it is 👍

Reply:
Ok. I'll try my hands on it again.